wrappers/android/zxingcpp/src/main/AndroidManifest.xml

```diff
@@ -1,2 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The empty manifest is not required now that namespace is declared in the `build.gradle.kts` but happy to restore this if desired.